CMakeCommon.cmake

```diff
@@ -122,6 +122,7 @@ function (GenerateAddOnProject acVersion devKitDir addOnName addOnSourcesFolder
 
     set (ResourceObjectsDir ${CMAKE_BINARY_DIR}/ResourceObjects)
     set (ResourceStampFile "${ResourceObjectsDir}/AddOnResources.stamp")
+    set (AC_ADDON_NAME ${addOnName} CACHE STRING "Name of the Add-On" FORCE)
```

Comment:
This function already gets the add-on name as a parameter. Specifying a cache variable doesn't seem logical in this function. Why do we need this line?

Reply:
Yeah, this line is not needed anymore since we don't need to read the data from the cache (the config file is the single source of thruth).

Reply:
Fixed